plugins/parsers/dropwizard/parser.go

```diff
@@ -211,18 +214,22 @@ func (p *Parser) readDWMetrics(metricType string, dwms interface{}, metrics []te
 
 			measurementWithTags := measurementName
 			for tagName, tagValue := range tags {
-				tagKeyValue := fmt.Sprintf("%s=%s", tagName, tagValue)
+				tagKeyValue := fmt.Sprintf("%s=%s", keyEscaper.Replace(tagName), keyEscaper.Replace(tagValue))
```

Comment:
We should use `metrics.New(` to make the metrics, then we won't need to worry about escaping here, and it should be faster as well.

Reply:
I wouldn't want to loose the default functionality. When you don't provide a custom template of your own, then each dropwizard measurement name is treated as line protocol keys `measurement<,tag_set>`.

Reply:
Ah right, maybe in the future I'll add some functions that can handle this more elegantly... I'm making some changes to metrics this month so maybe I can include it.

Reply:
having a utility function to parse the keys part of a measurement sounds good! Don't hesitate to ping once there is something in place.